install.sh

```diff
@@ -22,7 +22,7 @@ rocBLAS build & installation helper script
       -o | --cov                 Set Tensile code_object_version (V2 or V3)
       -t | --test_local_path     Use a local path for Tensile instead of remote GIT repo
            --cpu_ref_lib         Specify library to use for CPU reference code in testing (blis or lapack)
-           --[no-]hip-clang      Whether to build library for amdgpu backend using hip-clang
+           --hip-clang           Whether to build library for amdgpu backend using hip-clang
```

Comment:
I would remove `--hip-clang` entirely from the flags and just accept and ignore or give error for old options. Removing the `--no-hip-clang` option while leaving `--hip-clang` makes no sense.

Reply:
Devops requested that we support the option temporarily to give them time to upgrade without failed builds.

Reply:
It should be cleaned up after they transition.

Reply:
Support both options until you remove both entirely. Don't remove `--no-hip-clang` while leaving `--hip-clang`. 

Change the default all you wish, but allow the user to specify one or the other explicitly until you remove the options entirely.

You can remove the options from the documentation if you remove both the positive and negative sense options, even if you still support them as hidden options. But listing one without the other is misleading.